docs/src/getstarted/hello-world.md

```diff
@@ -0,0 +1,111 @@
+
+```rust
+use solana_program::{
+    account_info::AccountInfo,
+    entrypoint,
+    entrypoint::ProgramResult,
+    pubkey::Pubkey,
+    msg,
+};
+```
+
+Every Solana program must define an `entrypoint` that tells the Solana runtime where to start executing your on chain code. Your program's [entrypoint](../developing/on-chain-programs/developing-rust#program-entrypoint) should provide a public function named `process_instruction`:
+
+```rust
+// declare and export the program's entrypoint
+entrypoint!(process_instruction);
+
+// program entrypoint's implementation
+pub fn process_instruction(
+    program_id: &Pubkey,
+    accounts: &[AccountInfo],
+    instruction_data: &[u8]
+) -> ProgramResult {
+    // log a message to the blockchain
+    msg!("Hello, world!");
+
+    // gracefully exit the program
+    Ok(())
+}
+```
+
+Every on chain program should return the `Ok` [result enum](https://doc.rust-lang.org/std/result/) with a value of `()`. This tells the Solana runtime that your program executed successfully without errors.
+
+Our program above will simply [log a message](../developing/on-chain-programs/debugging#logging) of "_Hello, world!_" to the blockchain cluster, then gracefully exit with `Ok(())`.
+
+## Build your program
+
+On the left sidebar, select the "**Build & Deploy**" tab. Next, click the "Build" button.
+
+If you look at the Playground's terminal, you should see your Solana program begin to compile. Once complete, you will see a success message.
+
+![](/img/quickstarts/solana-get-started-successful-build.png)
+
+> Note:
+> You may receive _warning_ when your program is compiled due to unused variables. Don't worry, these warning will not affect your build. They are due to our very simple program not using all the variables we declared in the `process_instruction` function.
+
+## Deploy your program
+
+You can click the "Deploy" button to deploy your first program to the Solana blockchain. Specifically to your selected cluster (e.g. Devnet, Testnet, etc).
+
+After each deployment, you will see your Playground Wallet balance change. By default, Solana Playground will automatically request SOL airdrops on your behalf to ensure your wallet has enough SOL to cover the cost of deployment.
+
+![](/img/quickstarts/solana-get-started-build-and-deploy.png)
+
+#### Get your program's public address (aka program id)
+
+When executing a program using [web3.js](../developing/clients/javascript-reference.md) or from [another Solana program](../developing/programming-model/calling-between-programs.md), you will need to provide the program id (aka public address of your program).
+
+Inside Solana Playground's **Build & Deploy** sidebar, you can find your program id under the **Program Credentials** dropdown.
+
+#### Congratulations!
+
+You have successfully setup, built, and deployed a Solana program using the Rust language directly in your browser.
```

Comment:
Should we have something that executes the instruction? Otherwise the user won't ever see the "hello world" logged and not have the "aha success!" moment

Reply:
Ideally, yes. But I think that would require adding another section to this page about setting up, building, and connecting a web3.js app to be able to right? Which we can certainly add, I just figured this was a bit lighter to get through

Reply:
imo we should close the loop on actually logging "Hello World" on the blockchain. The information required to use web3.js in a test shouldn't be too long, but is vital for completing the Hello World tutorial.

At a later date playground will have support for writing JS and Rust tests and we'll be able to leverage that.